src/lib/plot/axis/axis.js

```diff
@@ -140,18 +145,20 @@ class Axis extends PureRenderComponent {
     };
 
     const {
-      left,
-      top,
-      width,
+      className,
       height,
+      left,
       orientation,
-      title
+      title,
+      top,
+      width
     } = props;
-
+    const isY = [LEFT, RIGHT].indexOf(orientation) > -1;
```

Comment:
I'm totally with you, but there aren't any examples of includes in react-vis, while there are a number of instances of indexOf